imagersite/imager_images/tests.py

```diff
@@ -29,54 +31,471 @@ class Meta:
+        super(TestCase, cls)
+        cls.username = 'person'
+        cls.password = 'password'
+        cls.testuser = models.User.objects.create_user(
+            username=cls.username,
+            password=cls.password
+        )
+        cls.testphoto = Photos.objects.create(
+            user=cls.testuser
+        )
 
     def test_photo_belongs_to_unique_user(self):
-        assert self.user1_photo.user.username == 'user1'
+        self.assertEqual(self.testphoto.user.username, self.username)
 
     def test_new_empty_description(self):
-        assert self.user1_photo.description == ''
+        self.assertEqual(self.testphoto.description, '')
 
     def test_new_empty_title(self):
-        assert self.user1_photo.title == ''
+        self.assertEqual(self.testphoto.title, '')
 
     def test_photo_access(self):
-        assert self.user1_photo.published == 'private'
-        self.user1_photo.published = 'public'
-        assert self.user1_photo.published == 'public'
+        self.assertEqual(self.testphoto.published, 'private')
 
     @classmethod
     def tearDownClass(cls):
-        users = User.objects.all()
-        non_super = users[1:]
-        for u in non_super:
-            u.delete()
+        super(TestCase, cls)
+        models.Photos.objects.all().delete()
+        models.User.objects.all().delete()
 
 
 class TestAlbum(TestCase):
     @classmethod
     def setUpClass(cls):
-        cls.user1_photo = PhotoFactory.create()
-        cls.user1_photo = PhotoFactory.create()
-        cls.user1_album = AlbumFactory.create()
+        super(TestCase, cls)
+        cls.username = 'person'
+        cls.password = 'password'
+        cls.testuser = models.User.objects.create_user(
+            username=cls.username,
+            password=cls.password
+        )
+        cls.testalbum = Album.objects.create(
+            user=cls.testuser
+        )
 
     def test_user_album_exists(self):
-        assert self.user1_album.user
+        self.assertIsNotNone(self.testalbum.user)
 
     def test_new_empty_title(self):
-        assert self.user1_album.title == ''
+        self.assertEqual(self.testalbum.title, '')
 
     def test_new_empty_desc(self):
-        assert self.user1_album.description == ''
+        self.assertEqual(self.testalbum.description, '')
 
-    def test_cover(self):
-        self.user1_album.cover(self.user1_photo)
-        assert self.user1_album == self.user1_photo
+    @classmethod
+    def tearDownClass(cls):
+        super(TestCase, cls)
+        models.Album.objects.all().delete()
+        models.User.objects.all().delete()
+
+
+class TestAlbumListView(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestCase, cls)
+        cls.username = 'person'
+        cls.password = 'password'
+        cls.testuser = models.User.objects.create_user(
+            username=cls.username,
+            password=cls.password
+        )
+        cls.c = Client()
```

Comment:
There's a drawback to instantiating the client here in the setUpClass method.  It means that the client's state will be shared across tests.  That's not often what you want.  Django's TestCase should give you a `self.client` out of the box.  Is there a reason that one wasn't good enough?